build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/shared-configuration.kt

```diff
@@ -179,16 +179,16 @@ fun Project.createTestTask(name: String, executer: String, sourceSet: SourceSet,
 
 private
 fun IntegrationTest.setUpAgentIfNeeded(testType: TestType, executer: String) {
+    if (executer == "embedded") {
```

Comment:
🎨 I think I could have used a comment here saying something like _Add the java agents to the test JVM when running integration tests using the embedded Gradle executer._